docs/source/notes/autograd.rst

```diff
@@ -849,9 +849,10 @@ Backward Hooks execution
 
 This section will discuss when different hooks fire or don't fire.
 Then it will discuss the order in which they are fired.
-The hooks that will be covered are: hooks registered to Tensor via
-:meth:`torch.tensor.register_hook`,
-post-hooks registered to Node via :meth:`torch.autograd.graph.Node.register_hook`, and
+The hooks that will be covered are: backward hooks registered to Tensor via
+:meth:`torch.tensor.register_hook`, post accumulate grad hooks registered to
```

Comment:
When referring to "post accumulate grad hooks" I feel like we may need to hyphenate it like "post-accumulate-grad hooks"